cookie-banner-rules-list.json

```diff
@@ -893,6 +893,1279 @@
+        "optIn": "button#didomi-notice-agree-button",
+        "presence": "div#buttons"
+      },
+      "domain": "jutarnji.hr",
+      "cookies": {},
+      "id": "fd22bda9-14dc-4777-92f0-971d474d4ee7"
+    },
+    {
+      "click": {
+        "optIn": "button#almacmp-modalConfirmBtn",
+        "presence": "div.almacmp-controls"
+      },
+      "domain": "iltalehti.fi",
+      "cookies": {},
+      "id": "23fcab4a-3b43-4d73-bb9e-57d86121141b"
+    },
+    {
+      "click": {
+        "optIn": "button#didomi-notice-agree-button",
+        "presence": "div#buttons"
+      },
+      "domain": "orange.fr",
+      "cookies": {},
+      "id": "dd60c9fc-2800-45b1-abd7-2ec401ff68b5"
+    },
+    {
+      "click": {
+        "optIn": "button#L2AGLb",
+        "optOut": "button#W0wltc",
+        "presence": "div.spoKVd"
+      },
+      "domain": "google.gr",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "NID",
+            "value": "511=ntCDgWUxji_8A64T5CnZHtmT3V1Yq1JP5oZ8v53ESF-h3ChgFwz2ZbGdE9imzdwDbDntCjOL4ZHQEx73LfPoX97VroPzNlyoojCvvHkDUpuOjfsWhwBtP9yWYAzTAhowsjCyxRrNJ_PHk2aql_LJMFTZq46iY_WBNk3c7qsBV54"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "SOCS",
+            "value": "CAESHAgBEhJnd3NfMjAyMjA5MjktMF9SQzEaAnJvIAEaBgiAkvOZBg"
+          }
+        ]
+      },
+      "id": "029b3695-6b57-423e-8d7d-78abfba94f51"
+    },
+    {
+      "click": {
+        "optIn": "button#L2AGLb",
+        "optOut": "button#W0wltc",
+        "presence": "div.spoKVd"
+      },
+      "domain": "google.hu",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "NID",
+            "value": "511=KQduV0v0HsjPa51Z2vqPopH-vPt9aCzt2MoaoC-Jm8lfmMGX52PfpZkIThxcRpHO1EWXQ3Xzk1xkDOH5b3r-w6bI6_8PsL5nI0Jp1T3hWmKXxOHKoLcnLOBOMjRUMcjt9OgQ30hypa9AAs-CGKnf71AtaKqQQR9IHmxI6ba-TdQ"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "SOCS",
+            "value": "CAESHAgBEhJnd3NfMjAyMjA5MjktMF9SQzEaAnJvIAEaBgiAkvOZBg"
+          }
+        ]
+      },
+      "id": "682598ca-1742-4689-ad95-7c70f2a01511"
+    },
+    {
+      "click": {
+        "optIn": "button#L2AGLb",
+        "optOut": "button#W0wltc",
+        "presence": "div.spoKVd"
+      },
+      "domain": "google.nl",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "NID",
+            "value": "511=XfZZs0qDp4KidKSqZVhkko77b-yy8Wi5cSAij6J_hHpQT9l-6dihYcjiEiZXiJi5dbsv6yCep3-hBxB072tJPyJbCpmXTeuRDO2KEzaXLP_XSux5rhv6Z0YmedxNLGD1ghe54L_fDvPvYwUTTe8AuJXPqc5YNGLkctL8jWiJA0o"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "SOCS",
+            "value": "CAESHAgBEhJnd3NfMjAyMjA5MjktMF9SQzEaAnJvIAEaBgiAkvOZBg"
+          }
+        ]
+      },
+      "id": "e3af03b3-8990-45ef-abd5-f6da31bd31fe"
+    },
+    {
+      "click": {
+        "optIn": "button.cmp-intro_acceptAll",
+        "presence": "div.cmp-intro_options"
+      },
+      "domain": "onet.pl",
```

Comment:
No banner with feature disabled (with German origin IP)

Reply:
We've looked over orange.fr and onet.pl and both have cookie banners for us on both Firefox and other browsers. Unsure why they wouldn't trigger for you in Germany.

Reply:
As long as you see banners and we handle them correctly that's ok. If no banner is shown our mechanism will simply not do anything.